view/src/components/account.js

```diff
@@ -0,0 +1,348 @@
+// account.js
+
+import React, { Component } from "react";
+
+import withStyles from "@material-ui/core/styles/withStyles";
+import Typography from "@material-ui/core/Typography";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import CloudUploadIcon from "@material-ui/icons/CloudUpload";
+import {
+  Card,
+  CardActions,
+  CardContent,
+  Divider,
+  Button,
+  Grid,
+  TextField,
+} from "@material-ui/core";
+
+import clsx from "clsx";
+
+import axios from "axios";
+import { authMiddleWare } from "../util/auth";
+
+const styles = (theme) => ({
+  content: {
+    flexGrow: 1,
+    padding: theme.spacing(3),
+  },
+  toolbar: theme.mixins.toolbar,
+  root: {},
+  details: {
+    display: "flex",
+  },
+  avatar: {
+    height: 110,
+    width: 100,
+    flexShrink: 0,
+    flexGrow: 0,
+  },
+  locationText: {
+    paddingLeft: "15px",
```

Comment:
nit: designers all ways tell me to do multiples of 4 for pretty much everything (there are exceptions ofc), probably not a bad idea to adjust things to follow that.